code/gradle.properties

```diff
@@ -16,7 +16,7 @@ org.gradle.caching=true
 android.useAndroidX=true
 
 moduleName=messaging
-moduleVersion=3.2.1
+moduleVersion=3.3.0
```

Comment:
nit: can we do this in a separate PR. Since we are dealing with snapshots it would be clearer have a good demarcation. Also, I think you can run the update_versions actions on this branch also...

Reply:
Done